src/SLCore/Service/Lifecycle/Models/FeatureFlagsDto.cs

```diff
@@ -28,5 +28,6 @@ public record FeatureFlagsDto(
         bool shouldManageServerSentEvents,
         bool enableDataflowBugDetection,
         bool shouldManageFullSynchronization,
-        bool enableTelemetry);
+        bool enableTelemetry,
```

Comment:
We are missing the first feature flag "shouldManageSmartNotifications":
https://github.com/SonarSource/sonarlint-core/blob/55a175cddc96ad0cc015ec8ad13e7c102d7724b3/rpc-protocol/src/main/java/org/sonarsource/sonarlint/core/rpc/protocol/backend/initialize/FeatureFlagsDto.java

Reply:
It will be handled by https://sonarsource.atlassian.net/browse/SLVS-1486